identity/src/main/java/com/android/identity/credentialtype/DrivingLicense.kt

```diff
@@ -0,0 +1,857 @@
+ * limitations under the License.
+ */
+
+package com.android.identity.credentialtype
+
+/**
+ * Object containing the metadata of the Driving License Credential Type
+ */
+object DrivingLicense {
+    const val MDL_NAMESPACE = "org.iso.18013.5.1"
+    const val AAMVA_NAMESPACE = "org.iso.18013.5.1.aamva"
+
+    /**
+     * Build the Driving License Credential Type
+     */
+    fun getCredentialType(): CredentialType {
+        return CredentialType.Builder("Driving License")
+            .addMdocCredentialType("org.iso.18013.5.1.mDL")
+            .addVcCredentialType("Iso18013DriversLicenseCredential")/*
+             * First the attributes that the mDL and VC Credential Type have in common
+             */
+            .addAttribute(
+                CredentialAttributeType.STRING,
+                "family_name",
+                "Family name",
+                "Last name, surname, or primary identifier, of the mDL holder.",
+                true,
+                MDL_NAMESPACE
+            )
+            .addAttribute(
+                CredentialAttributeType.STRING,
+                "given_name",
+                "Given names",
+                "First name(s), other name(s), or secondary identifier, of the mDL holder",
+                true,
+                MDL_NAMESPACE
+            )
+            .addAttribute(
+                CredentialAttributeType.DATE,
+                "birth_date",
+                "Date of birth",
+                "Day, month and year on which the mDL holder was born. If unknown, approximate date of birth",
+                true,
+                MDL_NAMESPACE
+            )
+            .addAttribute(
+                CredentialAttributeType.DATE,
+                "issue_date",
+                "Date of issue",
+                "Date when mDL was issued",
+                true,
+                MDL_NAMESPACE
+            )
+            .addAttribute(
+                CredentialAttributeType.DATE,
+                "expiry_date",
+                "Date of expiry",
+                "Date when mDL expires",
+                true,
+                MDL_NAMESPACE
+            )
+            .addAttribute(
+                CredentialAttributeType.StringOptions(Options.COUNTRY_ISO_3166_1_ALPHA_2),
+                "issuing_country",
+                "Issuing country",
+                "Alpha-2 country code, as defined in ISO 3166-1, of the issuing authority’s country or territory",
+                true,
+                MDL_NAMESPACE
+            )
+            .addAttribute(
+                CredentialAttributeType.STRING,
+                "issuing_authority",
+                "Issuing authority",
+                "Issuing authority name.",
+                true,
+                MDL_NAMESPACE
+            )
+            .addAttribute(
+                CredentialAttributeType.STRING,
+                "document_number",
+                "Licence number",
```

Comment:
In general we use US English in the source so this should be "License number" ("s", not "c"). Same in other data elements, for example "color" not "colour".

Reply:
Changed this, was just a copy of the specs...